test/data/testvib.py

```diff
@@ -69,6 +69,8 @@ def testirintens(self):
     @skipForParser('Molpro', 'Molpro cannot print force constants')
     @skipForParser('ORCA', 'ORCA cannot print force constants')
     @skipForParser('Turbomole', 'Turbomole cannot print force constants')
+    @skipForLogfile('Jaguar/Jaguar4.2', 'Data file does not contain force constants')
+    @skipForLogfile('Psi4/Psi4-1.0', 'Data file contains vibrational info with cartesian coordinates')

```

Comment:
Skip some logfiles in `data/regression`. Psi4-1.0 contains vibrational info with cartesian coordinates, instead of normal coordinates.